m365/powerbi/README.md

```diff
@@ -0,0 +1,116 @@
+## **SCuBA PowerBI Integration Steps**
+
+### **Supported Scenarios**
+
+This guide covers **two primary scenarios**:
+
+1. **PowerBI with Virtual Network Gateway:**
+   - Configure PowerBI to use a Virtual Network Gateway to access data from the ScubaConnect storage account.
+
+2. **PowerBI Desktop Direct Access:**
+   - Use PowerBI Desktop to retrieve and publish data directly to a PowerBI workspace. If the report needs to be viewed by others this is the ideal option. If others don't need to view the report, see below.
+   - You could also keep the data local and not publish to a PowerBI workspace.
```

Comment:
It took me a while to understand these options. I believe both are referring to a web-based PowerBI workspace and you can either set up the vnet gateway or manually push through the desktop app?

Considering we still plan to require the vnet in the near future, I'd consider changing these scenarios to "Automated Workspace Updates (Applicable to Commercial and GCC High" and "Manual Workspace Updates (GCC)"